doc/libpmemset/pmemset_source_from_file.3.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: manual
+Content-Style: 'text/css'
+title: _MP(PMEMSET_SOURCE_FROM_FILE, 3)
+collection: libpmemset
+header: PMDK
+date: pmemset API version 1.0
+...
+
+[comment]: <> (SPDX-License-Identifier: BSD-3-Clause)
+[comment]: <> (Copyright 2020, Intel Corporation)
+
+[comment]: <> (pmemset_source_from_file.3 -- man page for pmemset_source_from_file)
+
+[NAME](#name)<br />
+[SYNOPSIS](#synopsis)<br />
+[DESCRIPTION](#description)<br />
+[RETURN VALUE](#return-value)<br />
+[ERRORS](#errors)<br />
+[SEE ALSO](#see-also)<br />
+
+# NAME #
+
+**pmemset_source_from_file**() - creates an instance of persistent memory data source
+
+# SYNOPSIS #
+
+```c
+#include <libpmemset.h>
+
+int pmemset_source_from_file(struct pmemset_source **src, const char *file);
+```
+
+# DESCRIPTION #
+
+**pmemset_source_from_file**() function instantiates a new *struct pmemset_source** object
+describing the data source and sets a path to the file in it. Existence of the file indicated
+by the path provided to the data source is checked when calling **pmemset_part_new**() function.
```

Comment:
> Existence of the file indicated
by the path provided to the data source is checked when calling **pmemset_part_new**() function.

I think this is unnecessary. It can be described in the pmemset_part_new manpage.
But it is ok to mention about pmemset_part_new.